internal/guest/linux/ioctl.go

```diff
@@ -0,0 +1,36 @@
+//go:build linux
+// +build linux
+
+package linux
+
+import (
+	"os"
+
+	"golang.org/x/sys/unix"
+)
+
+const (
```

Comment:
Think it would be helpful to add a comment explaining that these constants are used to assemble the final ioctl command code passed. When reading this I put it together like this in my head:

```
0-7: NR
8-15: TYPE
16-29: SIZE
30-31: DIR
```